test/conftest.py

```diff
@@ -232,3 +235,26 @@ def wrapped() -> None:
             session.commit()
 
     return wrapped
+
+
+@fixture
+def validate_json():
+    def wrapped(data: dict, schema_name: str, schema_dir: str = ''):
+        # option to provide a schema_dir allows dummy_schema to be found for tests,
+        # this whole function will be removed and replaced with
+        # api-validator-python functionality
+        schema_dir = schema_dir or SCHEMA_DIRECTORY
+
+        schema_path = os.path.join(schema_dir, '{}.json'.format(schema_name))
+
+        try:
+            with open(schema_path) as schema_file:
+                validate(data, schema=json.load(schema_file))
+            return True
+        except (FileNotFoundError, ValidationError):
+            # Need to re raise with schema/validation failure information,
+            # though as this will be replaced by api-validator-python
+            # leaving for now
```

Comment:
Can this be replaced now @seanwiseman?

Reply:
Yes, but the tests using it would need amending slightly as the `JSONResponseValidator` takes the HTTP response object as it takes into account the media type etc. I left validating `json` to the `jsonschema` lib as not to just write a wrapper around the func.

I would leave for another PR.

Reply:
I'll revert change to reduce the diff here since it can be replaced anyway.